docs/source/reference/index.rst

```diff
@@ -9,10 +9,9 @@ Communicators
 
 .. autofunction:: create_communicator
 .. autoclass:: CommunicatorBase
-    :members: rank, intra_rank, inter_rank, inter_size, size,
-              alltoall, split, send, recv, bcast, gather, allreduce,
-              send_obj, recv_obj, bcast_obj, gather_obj,
-              allreduce_obj, bcast_data, allreduce_grad
+    :members: rank, intra_rank, size, alltoall, split, send, recv,
+              bcast, gather, allreduce, send_obj, recv_obj, bcast_obj,
+              gather_obj, allreduce_obj, bcast_data, allreduce_grad
```

Comment:
I don't know why this change happens but this is a rollback of https://github.com/chainer/chainermn/commit/f6c21a9e8d6164bac8c86b36c432530fa7356235 and should not happen.